ui/components/AgentDetail.tsx

```
@@ -0,0 +1,166 @@
+
+  return (
+    <div className="bg-white border border-beige-300 rounded-lg p-4 space-y-3">
+      <div className="font-medium text-beige-900">{agent.name}</div>
+      <div className="text-sm text-beige-600">{agent.handle}</div>
+
+      {/* Metadata Section */}
+      <div>
+        <button
+          onClick={() => toggleSection('metadata')}
+          className="w-full text-left flex items-center justify-between p-2 hover:bg-beige-100 rounded transition-colors"
+        >
```

Biome (2.1.2)

[error] Provide an explicit type prop for the button element.

The default type of a button is submit, which causes the submission of a form when placed inside a `form` element. This is likely not the behaviour that you want inside a React application.
Allowed button types are: submit, button or reset

(lint/a11y/useButtonType)